okhttp/src/main/java/okhttp3/internal/cache/DiskLruCache.kt

```diff
@@ -708,6 +708,8 @@ class DiskLruCache internal constructor(
 
           while (delegate.hasNext()) {
             val entry = delegate.next()
+            if (entry == null || !entry.readable) continue // Entry during edit
```

Comment:
I think this null check is unnecessary?

Reply:
I'll cleanup.  And then cherry pick onto the other branches.